src/storage/catalog_class.c

```diff
@@ -1315,12 +1315,18 @@ catcls_get_or_value_from_attribute (THREAD_ENTRY * thread_p, OR_BUF * buf_p, OR_
       goto error;
     }
 
-  /* flag */
+  /* Not Null */
```

Comment:
```suggestion
  /* flags */
```
해당 주석은 `META_ATTRIBUTE att_atts` 의 속성명을 나타낸 것으로 flags입니다.

Reply:
시스템 카탈로그 _db_attribute에서는 Not Null 관련 컬럼인 is_nullable과 flags 컬럼이 분리되어 있습니다.

여기 있는 Not Null 주석은 해당 컬럼인 &attrs[7].value를 나타내기 위해 쓰여있습니다.

Reply:
위에 있는 주석을 확인해 보시면 (id, type, ...) 시스템 카탈로그 기준이 아니라 현재 buf_p에 담겨있는 것을 기준으로 작성한 것입니다.

Reply:
주석 구분을 잘못했던 것 같습니다. 밑의 코멘트와 함께 반영하도록 하겠습니다. 감사합니다.